Makefile

```diff
@@ -1,18 +1,61 @@
-TARGET_DIR?=$(shell pwd)
-GITHUB_GROUP=github.com/th2-net
+SRC_MAIN_PROTO_DIR=src/main/proto
+GITHUB_TH2=github.com/th2-net
 
 TH2_GRPC_COMMON=th2-grpc-common
-TH2_GRPC_COMMON_URL=$(GITHUB_GROUP)/$(TH2_GRPC_COMMON)@makefile
+TH2_GRPC_COMMON_URL=$(GITHUB_TH2)/$(TH2_GRPC_COMMON)@makefile
 
-clean-deps:
-	-rm go.work
+MODULE_NAME=th2-grpc
+MODULE_DIR=$(MODULE_NAME)
 
-deps: clean-deps 
-	go work init .
+PROTOBUF_VERSION=v1.5.2
 
-	go get -u -t $(TH2_GRPC_COMMON_URL)
-	sleep 1
-	@cd $(shell go list -m -f '{{.Dir}}' $(TH2_GRPC_COMMON_URL)) && make generate-module TARGET_DIR=$(TARGET_DIR)
+PROTOC_VERSION=21.12
 
-test:
+default: test
```

Comment:
Please stay instructions for generation go code from proto and remove other instructions